Segment-Intercom/Classes/SEGIntercomIntegration.m

```diff
@@ -128,8 +135,8 @@ - (void)group:(SEGGroupPayload *)payload
 
 - (void)reset
 {
-    [self.intercom reset];
-    SEGLog(@" [Intercom reset];");
+    [self.intercom logout];
+    SEGLog(@" [Intercom logout];");
```

Comment:
what is the difference between logout and reset?

Reply:
reset is deprecated so they said to start using logout /shrug